lib/api/api.js

```diff
@@ -6,6 +6,7 @@ import bucketDelete from './bucketDelete';
 import bucketGet from './bucketGet';
 import bucketGetACL from './bucketGetACL';
 import bucketGetVersioning from './bucketGetVersioning';
+import bucketGetWebsite from './bucketGetWebsite';
```

Comment:
you should note that this pr depends on the one in arsenal adding this api to the request context.  please also write some tests in integration to test putBucketWebsite and getBucketWebsite by a user with a policy.